src/display/lv_display.c

```diff
@@ -405,7 +405,15 @@ void lv_display_set_buffers(lv_display_t * disp, void * buf1, void * buf2, uint3
 
     lv_color_format_t cf = lv_display_get_color_format(disp);
     uint32_t stride = lv_draw_buf_width_to_stride(w, cf);
-    if(stride * h > buf_size && render_mode != LV_DISPLAY_RENDER_MODE_PARTIAL) {
+    if(LV_DISPLAY_RENDER_MODE_PARTIAL == render_mode) {
+        h = buf_size / stride;
+        if(0 == h) {
+            LV_LOG_ERROR("the buffer is too small for PARTIAL mode");
+            LV_ASSERT(0);
+            return;
```

Comment:
uint32_t min_h = buf_size / stride;
LV_ASSERT_MSG(min_h != 0, "the buffer is too small for PARTIAL mode");

Reply:
Parameter `h` must be updated too. Change assert message to `> 0`
```
h = buf_size / stride;
LV_ASSERT_MSG(h > 0, "the buffer is too small for PARTIAL mode");
```

Reply:
If we use min_h, how could we update the h?